packages/eslint-plugin-shopify/lib/config/rules/flowtype.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
   'flowtype/require-parameter-type': 'warn',
   // Requires that functions have return type annotation.
   'flowtype/require-return-type': ['warn', 'always', {annotateUndefined: 'never'}],
+  // Makes sure that files have a valid @flow annotation.
+  'flowtype/require-valid-file-annotation': 'warn',
```

Comment:
Good call. I think I am also going to turn off the required types for return/ parameter, generally I think inference is better and less overbearing. Let me know if you think that's a bad idea.